pkgs/tools/inputmethods/skk/skk-dicts/default.nix

```diff
@@ -1,77 +1,192 @@
-{ lib, stdenv, fetchurl, iconv, skktools }:
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, nix-update-script
+, nkf
+, useUtf8 ? false
```

Comment:
I made UTF-8 conversion of the dictionary files opt-in because it deviates from the vanilla experience. It might be beneficial to expose a top level `skkDictionariesUtf8` attribute if packages become to depend exclusively on UTF-8 dictionaries.